src/test/__tests__/ReactDefaultPerf-test.js

```diff
@@ -24,7 +24,7 @@ describe('ReactDefaultPerf', function() {
 
   beforeEach(function() {
     var now = 0;
-    jest.setMock('performanceNow', function() {
+    jest.setMock('fbjs/lib/performanceNow', function() {
```

Comment:
Oh, `setMock` isn't [one of the methods](https://github.com/facebook/fbjs/blob/master/scripts/babel/rewrite-modules.js#L84-L86) that the transform catches 😢. Will fix that.